tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
```

Comment:
"As JSON is based on a very restricted subset of JavaScript syntax, trailing commas are not allowed in JSON."
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas